graphql/directive_test.go

```diff
@@ -891,3 +891,78 @@ func TestRequirePatchOwner(t *testing.T) {
 		})
 	}
 }
+func TestRequireAdmin(t *testing.T) {
+	defer func() {
+		err := db.ClearCollections(user.Collection)
+		require.NoError(t, err, "unable to clear user collection")
+	}()
+
+	obj := any(nil)
+
+	for tName, tCase := range map[string]func(ctx context.Context, t *testing.T, config Config, usr *user.DBUser){
+		"SucceedsWhenUserIsAdmin": func(ctx context.Context, t *testing.T, config Config, usr *user.DBUser) {
+			role := "superuser"
+			require.NoError(t, usr.AddRole(ctx, role), "Failed to add role")
+			defer func() {
+				require.NoError(t, usr.RemoveRole(ctx, role), "Failed to remove role")
+			}()
```

Comment:
nit. you don't need to remove role in each function if you call
```
   require.NoError(t, db.ClearCollections(user.Collection))
```
in the set up portion